recipes/libcpucycles/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -ex
+
+echo "$CC $CFLAGS" >> compilers/default
+
+./configure --prefix=$PREFIX
+
+make
+
```

Comment:
Is there a `make check` or `make test` step we can run?

Reply:
As far as I could see, there isn't.